exporter/containerimage/writer.go

```diff
@@ -200,7 +202,7 @@ func (ic *ImageWriter) Commit(ctx context.Context, inp *exporter.Source, session
 			}
 		}
 
-		desc, _, err := ic.commitDistributionManifest(ctx, opts, r, config, remote, opts.Annotations.Platform(&p.Platform), inlineCache, opts.Epoch, session.NewGroup(sessionID))
+		desc, _, err := ic.commitDistributionManifest(ctx, opts, r, config, remote, opts.Annotations.Platform(&p.Platform), inlineCache, opts.Epoch, opts.RewriteEpoch, session.NewGroup(sessionID))
```

Comment:
As discussed in slack, I don't understand these conversions away from `Remote` as I think `Remote` can just be converted on its own and it can be done outside the `commitDistributionManifest` call.

Reply:
done